src/Features/ERDDAP/Platform/Forecasts/Page/story-data.ts

```diff
@@ -0,0 +1,25 @@
+import { platform } from "stories/platform"
+
+import { forecastToStandardNames } from "./index"
+
+export const forecast_type = "wave_height"
+
+export const standard_name = forecastToStandardNames[forecast_type]
+
+export const wave_datasets = platform.properties.readings.filter((reading) =>
+  standard_name.has(reading.data_type.standard_name)
+)
+
+export let data = wave_datasets.map((dataset) => ({
```

Comment:
Codacy found an issue: [Identifier 'wave_datasets' is not in camel case.](https://app.codacy.com/manual/gmri/Neracoos-1-Buoy-App/pullRequest?prid=6071156)